rt/rs/client/src/main/java/org/apache/cxf/jaxrs/client/spec/ClientImpl.java

```diff
@@ -52,8 +52,8 @@
 import org.apache.cxf.transport.https.SSLUtils;
 
 public class ClientImpl implements Client {
-    static final String HTTP_CONNECTION_TIMEOUT_PROP = "http.connection.timeout";
-    static final String HTTP_RECEIVE_TIMEOUT_PROP = "http.receive.timeout";
+    public static final String HTTP_CONNECTION_TIMEOUT_PROP = "http.connection.timeout";
```

Comment:
May be we could extract these props to `ClientProperties` since they are used in quite a few places?

Reply:
Does `ClientProperties` already exist, or are you suggesting I create a new class for it?